.vscode/custom-wordlist.txt

```diff
@@ -0,0 +1,32 @@
+ASAN
+Barbehenn
+bigoinput
+cbtf
+chegg
+CRAN
+Dalpiaz
+DSPM
+dygraphs
+Eddelbuettel
+entifier
+gamification
+Ihantola
+incentivized
```

Comment:
Not sure what dictionary they are using for stuff like this to not be picked up.